compiler/rustc_typeck/src/check/pat.rs

```diff
@@ -1404,6 +1404,7 @@ impl<'a, 'tcx> FnCtxt<'a, 'tcx> {
         )
         .span_label(span, format!("multiple uses of `{}` in pattern", ident))
         .span_label(other_field, format!("first use of `{}`", ident))
+        .span_suggestion(span, "consider removing one usage of", ident.to_string(), rustc_errors::Applicability::MachineApplicable)
```

Comment:
I would probably drop this one: we have no way of figuring out which of the uses was likely intended.